xml_inscriptions/bib_only/MO.Col.UM.MAA.L.83.267.xml

```diff
@@ -56,8 +98,12 @@
       <body>
          <div type="edition">
          <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
+            <ab>
+               <lb n="1"/>Ex <expan><abbr>pr</abbr><ex>aediis</ex></expan> <persName><name type="gentilicium" key="Claudius">Cl<unclear>a</unclear><supplied reason="lost" evidence="parallel">udi</supplied></name> <supplied reason="lost" evidence="parallel"><name type="cognomen" key="Maximus">Maximi</name></supplied></persName>
+               <lb n="2"/>ab <persName key="Isis" type="slaveName"></persName>.
```

Comment:
Name markup is off; should be `<persName><name key="Isis" type="slaveName">Iside</name></persName>` (I don't have an image handy so I might be wrong on the form of the name).

Reply:
I agree that it should be "Iside" but CIL has it as "Isis" and the image is a little unclear:
http://maacollections.missouri.edu/ArgusNET/Portal.aspx?lang=en-US&g_AAEM=83.267 
<img width="506" alt="Screen Shot 2021-08-10 at 11 47 57 AM" src="https://user-images.githubusercontent.com/60520420/128837122-cb5f39d8-bc6e-4aa0-a433-1b28fcd78feb.png">

Should I leave a note in the file or ask Prof. Bodel about it?

Reply:
Might be safest to ask; the bigger issue is the empty element. Once you settle on what you think should be there, make sure to put it in.